index.js

```diff
@@ -113,6 +112,10 @@ var BrowserStackBrowser = function(id, emitter, args, logger,
              process.env.BUILD_TAG || process.env.CIRCLE_BUILD_NUM || null
     };
 
+    if (args.real_mobile) {
```

Comment:
I'd add a comment that this should always be a string `'true'` or `'false'` so just checking for thruthiness is enough. Other than that it LGTM.